src/output.cpp

```diff
@@ -52,12 +52,9 @@ extern bool test_mode;
 
 void delwin_functor::operator()( WINDOW *w ) const
 {
-    if( w == nullptr ) {
-        return;
+    if( w != nullptr ) {
+        delwin( w );
     }
-    werase( w );
-    wrefresh( w );
-    delwin( w );
```

Comment:
I doubt that this `werase/refresh` really matters and that something relies on this.
The game should either restore the underlying background or redraw it completely. Replacing it with a black rectangle is meaningless. The only outcome of this is #15486.